shared.proto

```diff
@@ -182,6 +182,8 @@ message SystemInformationAmazonRDS {
   bool parameter_auto_explain_enabled = 42;
 
   bool is_aurora_postgres = 50;
+
+  map<string, string> tags = 60;
```

Comment:
I think that works, though I wonder if it makes sense to store this as a more general "resource_tags" under the `SystemInformation` message?

e.g. if we add support for GCP, maybe its better if that re-uses the same field, so we don't to special case each provider in the processing of the messages?

Reply:
Good point, I've moved it over